.github/workflows/_Test-OCI-Factory.yaml

```diff
@@ -12,6 +12,7 @@ on:
       - "oci/mock*"
       - "src/**"
       - "!src/workflow-engine/**"
+      - "!src/cli-client/**"
```

Comment:
we need a new workflow just to run the unit tests though

Reply:
Added as ROCKS-1272 in JIRA.